### PR TITLE
Add customizable style classes

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?
Adds the classes to the item and menu component.

#### What problem is this solving?
Before, there was no way to customize the components with an outside stylesheet.

#### How should this be manually tested?
Visit the [workspace](https://lucas--storecomponents.myvtex.com/category-menu).

#### Screenshots or example usage
screen shot 2018-04-20 at 09 33 59
screen shot 2018-04-20 at 09 34 07

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/CategoryMenu.js [hunk 1]
```diff
@@ -11,11 +11,13 @@ import LoadingBar from './components/LoadingBar'
...
   render() {
-    const { data: { categories, loading } } = this.props
+    const {
+      data: { categories, loading },
+    } = this.props
 
     return (
       <LoadingBar loading={loading}>
-        <div className="h3 bg-near-white">
+        <div className="category-menu h3 bg-near-white">
```
Comment (ClaudivanFilho) [inline react/CategoryMenu.js]: falta o prefixo vtex. `vtex-category-menu`

File: react/__tests__/__snapshots__/CategoryItem.test.js.snap [hunk 2]
```diff
@@ -2,43 +2,34 @@
 
 exports[`CategoryItem component should match snapshot 1`] = `
 <div
-  className="h3 w4"
+  className="h3 w4 category-menu__item dib pl5 pr4 black-90 hover-bg-black-90 hover-white show-arrow"
```
Comment (brunojdo) [inline react/__tests__/__snapshots__/CategoryItem.test.js.snap]: vtex-category...  ?

File: react/package.json [hunk 3]
```diff
@@ -2,37 +2,21 @@
   "name": "vtex.render-getting-started",
   "version": "0.0.0",
   "scripts": {
-    "test": "jest"
+    "test": "vtex-scripts test"
   },
   "dependencies": {
     "@vtex/styleguide": "^2.0.0-rc.35",
```
Comment (brunojdo) [inline react/package.json]: 39

File: react/__tests__/__snapshots__/CategoryItem.test.js.snap [hunk 4]
```diff
@@ -2,43 +2,34 @@
 
 exports[`CategoryItem component should match snapshot 1`] = `
 <div
-  className="h3 w4"
+  className="h3 w4 vtex-category-menu__item dib pl5 pr4 black-90 hover-bg-black-90 hover-white show-arrow"
```
Comment (brunojdo) [inline react/__tests__/__snapshots__/CategoryItem.test.js.snap]: namespace first

File: react/components/CategoryItem.js [hunk 5]
```diff
@@ -13,47 +13,40 @@ class CategoryItem extends Component {
     const hasChildren = category.hasChildren || category.children.length > 0
 
     const wrapperClasses = classNames(
-      'h-100 w-100 dib pl5 pr4 vtex-category-item black-90 hover-bg-black-90 hover-white',
+      'h3 w4 vtex-category-menu__item dib pl5 pr4 black-90 hover-bg-black-90 hover-white',
```
Comment (brunojdo) [inline react/components/CategoryItem.js]: namespace first

File: react/theme.css [hunk 6]
```diff
@@ -30,15 +30,19 @@ body {
```
Comment (brunojdo) [inline react/theme.css]: See the msg that I send on slack.

**Pull request merged**